github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -3,7 +3,8 @@ presubmits:
   - name: check-up-kind-1.17-sriov
     annotations:
       k8s.v1.cni.cncf.io/networks: multus-cni-ns/sriov-passthrough-cni
-    always_run: true
+    always_run: false # only because we have run_if_changed
+    run_if_changed: "^(cluster-up/cluster/kind-k8s-sriov-1.17.0/.*)|(cluster-up/cluster/kind-k8s-sriov-1.17.0/certcreator/.*)|(cluster-up/cluster/kind-k8s-sriov-1.17.0/certcreator/certlib/.*)|(cluster-up/cluster/kind-k8s-sriov-1.17.0/manifests/.*)$"
```

Comment:
according community
`"^cluster-up/cluster/kind-k8s-sriov-1.17.0/.*"` should be enough to also catch files and folder recursively
Updating to that in order to make it simpler and less to maintain.